modules/twinklewarn.js

```diff
@@ -1644,16 +1475,18 @@ Twinkle.warn.callbacks = {
 		text += "|sig=true}}";
 		return text;
 	},
+	// NOTE: used by block module
 	preview: function(form) {
-		var templatename = form.sub_group.value;
-		var linkedarticle = form.article.value;
-		var templatetext;
-
-		if (templatename in Twinkle.warn.messages.block) {
-			templatetext = Twinkle.warn.callbacks.getBlockNoticeWikitext(templatename, linkedarticle, form.block_timer.value,
-				form.block_reason.value, Twinkle.warn.messages.block[templatename].indefinite);
+		var templatename = form.sub_group.value,
+				linkedarticle = form.article.value,
+				templatetext;
+
+		if (templatename in Twinkle.block.blockPresetsInfo) {
+			// form.expiry.value is for block module, where expiry is the key name that the API uses
```

Comment:
This is a little confusing, at first glance. Why do we need two similarly named but distinct keys?